js/widgets/rhythmruler.js

```diff
@@ -17,6 +17,34 @@
 // rulerButtonsDiv is for the widget buttons
 // rulerTableDiv is for the drum buttons (fixed first col) and the ruler cells
 
+function gcd(max,min){
```

Comment:
Maybe use the versions in utils/utils.js rather than redefining lcd and gcd here?